docker-compose.override.yml

```diff
@@ -6,14 +6,17 @@ services:
       context: .
       target: gulp
     command: bash -c "npx gulp assemble --environment=development --sass-lint=${LINT_SASS} && npx gulp watch --environment=development --sass-lint=${LINT_SASS}"
+    restart: on-failure
```

Comment:
This has always been one of the most overloaded options, but `on-failure` is correct for this use case https://docs.docker.com/config/containers/start-containers-automatically/#use-a-restart-policy